### DISCUSSION TITLE
q-table selected

### DISCUSSION THREAD

**Discussion by EstefanyEH** [quasarframework/quasar#14100] [General - Components / Directives / etc]
Hello everyone, please help me. When I select an item, the check is activated in all, I share with you the code
![Captura de Pantalla 2022-07-29 a la(s) 17 41 12](https://user-images.githubusercontent.com/87049170/181840340-d98026cd-32c9-4f47-80c0-d40b870b956b.png)
![Captura de Pantalla 2022-07-29 a la(s) 17 41 12 (2)](https://user-images.githubusercontent.com/87049170/181840347-115de5e8-beba-42af-ab58-54ad478a76f4.png)

**Comment by pratik227** [comment]
Try this.

https://codepen.io/Pratik__007/pen/eYMewMq

`      :row-key="row => row.name.value"
`

**Reply by EstefanyEH** [reply]
thanks, i finally re-organized the data i got from the api, in column and rows i set name and then in the key-row i used only name
